jujugui/static/gui/src/app/app.js

```diff
@@ -1086,6 +1086,10 @@ YUI.add('juju-gui', function(Y) {
         // as it's visible but not functional.
         showEnvSwitcher = false;
       }
+      // If we're in sandbox we don't want to display the switcher.
+      if (this.get('sandbox')) {
+        showEnvSwitcher = false;
+      }
```

Comment:
Readability and cleanliness. They're two different situations, so I wanted to keep them separate, with separate comments, which would make it easier should we ever change our minds on one or the other.